### DISCUSSION TITLE
🤓Advanced Custom CSS Options

### DISCUSSION THREAD

**Discussion by R0shOnE** [elementor/elementor#21654] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

More UI/UX options when using the CUSTOM CSS section

### Describe the solution you'd like

It would be nice to add some buttons/options to improve the CUSTOM CSS experience:

1. Adding a dynamic tag button searching into existing css variables (eq: colors, fonts, breakpoints..). 
2. The "selector" or "child-element" functions working good, but sometimes we have to "inspect" the exact class, so maybe adding a light built in inspector that on click can inspect inside the selector to copy the class. 

### Describe alternatives you've considered

1. alternative way, when tapping "var(" display all sorted variables?
2. alternative way, when tapping ".elementor-" display all class inside the selector?

**Comment by ColeKafkac** [comment]
Specifically, you'd like to see some new buttons or options added to make it easier to work with CSS variables and selectors.

One suggestion you've made is to add a dynamic tag button that searches existing CSS variables like colors, fonts, and breakpoints. Another idea is to add a light built-in inspector that allows you to inspect inside the selector and copy the class.

You've also considered some alternative ways to achieve these improvements, such as displaying all sorted variables when tapping "var(" or displaying all classes inside the selector when tapping ".elementor-".

These are great ideas that could definitely help improve the user experience of the CUSTOM CSS section in Elementor Pro. I suggest submitting your feature request to the Elementor Pro team via their official channels, such as their support forum or feature request form.

Thanks for sharing your ideas and contributing to the improvement of Elementor Pro! Let me know if you have any further questions or if there's anything else I can do to assist you.

**Reply by R0shOnE** [reply]
Hey, thank you I thought this place is the right place to suggest a feature...
